addOns/ascanrules/ascanrules.gradle.kts

```diff
@@ -13,7 +13,7 @@ zapAddOn {
         dependencies {
             addOns {
                 register("commonlib") {
-                    version.set(">= 1.20.0 & < 2.0.0")
+                    version.set(">= 1.21.0 & < 2.0.0")
```

Comment:
This does not seem to be needed, in any case worth mentioning in the changelog.

Reply:
It will be in order to ge the right version/update of vulnerabilities.xml won't it?

Reply:
Added to changelog.

Reply:
IMO the minimum should be only bumped for required binary changes which is not the case (just saying not asking to be reverted).